packages/contracts/src/util.ts

```diff
@@ -0,0 +1,65 @@
+import { ResultInterface } from '@blockstack/clarity'
+import { TextEncoder } from 'util';
+
+export namespace Clarity {
+  export function uint(val: number): string {
+    return `u${BigInt(val).toString()}`;
+  }
+
+  export function int(val: number): string {
+    return `${BigInt(val).toString()}`;
+  }
+
+  export function principal(val: string): string {
+    return `'${val}`;
+  }
+
+  export function stringBuff(val: string): string {
+    const encoder = new TextEncoder();
+    const buffer = encoder.encode(val)
+    return "0x" + Array.prototype.map.call(new Uint8Array(buffer), x => ('00' + x.toString(16)).slice(-2)).join('');
+  }
+
+  export function string(val: string): string {
+    return `"${val}"`;
```

Comment:
If we pass string as contract argument without using this weird black magic our unit tests will start failing like this:
```
ExecutionError: Eval expression on contract failed with bad exit code 1: Program execution error: 
Unchecked(UndefinedVariable("test-string"))
```
Because of that we have to wrap all strings with double quotes.

I think it has something to do with how contract calls are executed by official Stacks library, but I don't have enough knowledge to explain it in details.

Reply:
Thanks for explanation. When I get a chance I'll run this and check out the stack trace to try and get more of an understanding. I'd like to leave this unresolved just as a reminder for me to check it out.